src/db/models/User.ts

```diff
@@ -79,12 +81,12 @@ User.init({
     type: new DataTypes.STRING(128),
     allowNull: false,
   },
-  contactID: {
-    type: new DataTypes.STRING(128),
-    allowNull: true,
+  civility: {
+    type: new DataTypes.ENUM(...enumToStrArr(Civility)),
+    allowNull: false,
   },
-  contactType: {
-    type: new DataTypes.STRING(128),
+  displayEmail: {
+    type: new DataTypes.STRING(256),
     allowNull: true,
```

Comment:
I think it shouldnt be null, instead we should set by default its value = signup email.
This way, in the frontend they always refer to display email? Thats debatable

Reply:
Based on what Diane said, I think it's different: email is the PERSONAL one and dispaly email is the COMPANY branch generic one, owned by multiple users. Not sure...